packages/dev/core/src/Materials/material.ts

```diff
@@ -1189,7 +1183,7 @@ export class Material implements IAnimatable, IClipPlanesHolder {
     public _preBind(effect?: Effect | DrawWrapper, overrideOrientation: Nullable<number> = null): boolean {
         const engine = this._scene.getEngine();
 
-        const orientation = overrideOrientation == null ? this.sideOrientation : overrideOrientation;
+        const orientation = this.sideOrientation == null ? overrideOrientation : this.sideOrientation;

```

Comment:
Is this right? `overrideOrientation` can be null, so it needs to fall back like it was before?

Maybe this? Or do nothing.
```suggestion
        const orientation = overrideOrientation ?? this.sideOrientation;
```

Reply:
I had understood that it would only fall back to overrideOrientation if the side orientation was null? But I might have understood incorrectly.